clang/test/CIR/IR/call.cir

```diff
@@ -0,0 +1,17 @@
+// RUN: cir-opt %s | FileCheck %s
+
```

Comment:
Not a review item, more like a general "it will be nice comment": `mlir-opt` has a `--verify-roundtrip` flag, which we should add at some point to `cir-opt` to make such testings even more robust.